doc/productivity.md

```diff
@@ -93,6 +94,21 @@ To simply check out a commit-ish under a new working directory without disruptin
 git worktree add --checkout ../where-my-checkout-commit-ish-will-live my-checkout-commit-ish
 ```
 
+### Dummy rebases for fixups and execs with `git merge-base`
```

Comment:
I think this is called interactive rebase?
```suggestion
### Interactive rebases for fixups and execs with `git merge-base`
```

Reply:
Well, they're interactive dummy rebases... Interactive rebases are normally over an updated master if you give it `master` instead of `"$(git merge-base master HEAD)"`.

Reply:
Why would you do an interactive rebase if it is to rebase on an updated master?

dummy rebase returns no results for me btw: https://duckduckgo.com/?q=Dummy+rebase

Reply:
You're right, an interactive rebase is not needed at all on an un-updated master (I hope that's what you meant). I just thought it'd be nice for people to see what the flags do to the rebase TODO list and understand what's going on, but we can remove it for sure. "Dummy rebase" is not a proper term, but I've had a use for this trick quite often when working on my git tree so I thought to add it.

Reply:
I like the "last common commit" trick but this really is exactly what I think most people think "interactive rebases" are, including me :P

Reply:
@instagibbs Well I did think about just adding a "How to find the last common commit" trick, but it doesn't really make sense without these two motivating invocations that I find myself doing often... I can remove the `-i` if that makes it more clear?

Reply:
No it's clear, I just have never heard of "dummy rebase". I'm agreeing with @MarcoFalke

Reply:
I found the term "dummy rebase" confusing as well, but perhaps that's because almost all rebases I do don't actually change the base, so what you're calling a dummy rebase is just what I'd call a git rebase.

Reply:
> I found the term "dummy rebase" confusing as well, but perhaps that's because almost all rebases I do don't actually change the base, so what you're calling a dummy rebase is just what I'd call a git rebase.

Curious: how do you achieve "not changing the base" without using `git merge-base`? Maybe there's something simpler I'm missing.

Reply:
`git rebase -i @~<number_of_commits>`

Reply:
@dongcarl I just use `git rebase -i HEAD~~~~~~` with a sufficient number of ~s. Your suggested approach is much better, i was just wondering what dummy rebase referred to.